aiida/plugins/entry_point.py

```diff
@@ -27,6 +27,11 @@
 ENTRY_POINT_STRING_SEPARATOR = ':'
 
 
+@functools.lru_cache(maxsize=1)
```

Comment:
I prefer not to have mutable global variables, i.e. this instead of `EPS = eps()`

Reply:
is this the recommended way of doing non-mutable global variables?

Anyhow, it's ok with me